.github/workflows/rw-backup-postgres-database.yml

```diff
@@ -78,4 +78,5 @@ jobs:
             az storage blob upload \
             --account-name ${{ inputs.storage_account_name }} \
             --container-name ${{ inputs.storage_container_name }} \
-            --file ${{ inputs.backup_file_name }}
+            --file postgres_backup.sql \
+            --name ${{ inputs.backup_file_name }}
```

Comment:
There should be no changes here as the maria db backup does not use this pipeline. Please revert the change

Reply:
Maybe we added this for consistency. If it is correct then we can keep the change